### PR TITLE
Bakta data manager update value and dbkey

### PR DESCRIPTION
**Pull request by pimarin**
Juste update the DM to indictae the version, date for user instead of DOI of the database on zenodo

### CODE REVIEW

File: data_managers/data_manager_build_bakta_database/test-data/bakta_test.loc [hunk 1]
```diff
@@ -9,3 +9,9 @@
 7360139	V2.0_2022-11-25	1.5	/tmp/tmpxrkfnuec/galaxy-dev/tool-data/bakta_database/7360139
 7197299	V1.0_2022-10-12	1.4	/tmp/tmpwe9n4gyg/galaxy-dev/tool-data/bakta_database/7197299
 7360139	V2.0_2022-11-25	1.5	/tmp/tmpwe9n4gyg/galaxy-dev/tool-data/bakta_database/7360139
+V1.0_2022-10-12	7197299	1.4	/tmp/tmpw5w3lbzc/galaxy-dev/tool-data/bakta_database/V1.0_2022-10-12
```
Comment (bgruening) [inline data_managers/data_manager_build_bakta_database/test-data/bakta_test.loc:12]: You do not add any new information to the data table, do you? If you want to display the second column and not the first one you can simply change this in the tool using the data table. Or do I miss something?

Comment (bgruening) [reply data_managers/data_manager_build_bakta_database/test-data/bakta_test.loc:12]: I assume you only mixed the `value` and `dbkey` column descriptions, can this be?

Comment (bgruening) [reply data_managers/data_manager_build_bakta_database/test-data/bakta_test.loc:12]: > The text that is displayed in the browser's dropdown is what's in the name column, and the value that would be stored for the parameter is in the value column.

Comment (pimarin) [reply data_managers/data_manager_build_bakta_database/test-data/bakta_test.loc:12]: You right, I just change in the python file the value and dbkey output.

Comment (bgruening) [reply data_managers/data_manager_build_bakta_database/test-data/bakta_test.loc:12]: I think the trick is to have the columns named `name` and `value` and the `name` column is displayed to the end user - so should be human readable.

Thanks @pimarin

Comment (pimarin) [reply data_managers/data_manager_build_bakta_database/test-data/bakta_test.loc:12]: oky I'll  change it !

File: data_managers/data_manager_build_bakta_database/data_manager/bakta_build_database.py [hunk 2]
```diff
@@ -78,13 +78,13 @@ def fetch_db_versions(self, db_version="latest"):
 
     def get_data_manager(self, bakta_database_info):
         self.bakta_table_list = self.get_data_table_format()
-        bakta_value = f"V{bakta_database_info['major']}." \
+        bakta_name = f"V{bakta_database_info['major']}." \
```
Comment (oschwengers) [inline data_managers/data_manager_build_bakta_database/data_manager/bakta_build_database.py:81]: Flake throws an error due to a wrong indentation that was introduced by replacing `bakta_value` by `bakta_name`.

**Comment by pimarin** [comment]
is it okay ?

**Comment by pimarin** [comment]
is it okay ?

**Comment by pimarin** [comment]
@bernt-matthias ? is it okay ?

**Comment by pimarin** [comment]
@bgruening is it okay ?

**Review by bgruening — APPROVED** [review]

**Comment by bgruening** [comment]
Lets try!

**Comment by bgruening** [comment]
Thanks @pimarin!

**Comment by pimarin** [comment]
Thank you @bgruening !!

**Pull request merged**